src/components/DCLMusicInfo/DCLMusicInfo.tsx

```diff
@@ -152,58 +125,21 @@ const InfoItem = styled.div`
   gap: 8px;
```

Comment:
```suggestion
  gap: 8px;
  align-items: baseline;
```
current styles:
<img width="907" alt="image" src="https://github.com/user-attachments/assets/f7749ea5-026b-4475-b26b-9f25187b0aff">


using `align-items: baseline;`:
<img width="918" alt="image" src="https://github.com/user-attachments/assets/161d6612-9649-46a7-88c3-08e9f3309ecd">